src/main/java/uk/gov/justice/hmpps/prison/service/InmateAlertService.java

```diff
@@ -79,7 +79,7 @@ public Page<Alert> getInmateAlerts(final Long bookingId, final String orderBy, f
         return alerts;
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
empty override roles is now possible.  The only client using this is prison-staff-hub which does not have SYSTEM_USER and so we can remove the override